schedulearn-frontend/src/components/WorkerList/WorkerList.tsx

```diff
@@ -28,8 +28,8 @@ export default class WorkerList extends React.Component<WorkerListProps, {}> {
           <tr key={index}>
             <td>{item.FullName}</td>
             <td>{item.JobTitle}</td>
-            <td>{item.DateFrom}</td>
-            <td>{item.DateTo}</td>
+            <td>{item.DateFrom.replace("T", " ")}</td>
+            <td>{item.DateTo.replace("T", " ")}</td>
```

Comment:
Otherwise dates show up like this:
![image](https://user-images.githubusercontent.com/35849268/80392759-3c955100-88b8-11ea-9dab-2e268038fa20.png)

Reply:
This string needs to be converted to date and then formatted to format of choice (perhaps there is a method that formats by current locale?), because the DB will save UTC, which will be without a timezone.